Test/logger/ResCountXML.dfy

```diff
@@ -0,0 +1,41 @@
+// RUN: %dafny /xml:%t.xml %s"
```

Comment:
Isn't this just redundantly testing the Boogie `/xml` feature?

Reply:
It's checking that it specifically includes the resource count (which it didn't before) to ensure that tests fail with too old a Boogie version.